src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java

```diff
@@ -339,6 +339,16 @@ public void actionPerformed(final ActionEvent e) {
                 for (final Feature peak : row.getPeaks()) {
                     if (mzRange == null) {
                         mzRange = peak.getRawDataPointsMZRange();
+                        double upper = mzRange.upperEndpoint();
+                        double lower = mzRange.lowerEndpoint();
+                        if ((upper - lower) < 0.000001) {
+                            //Workaround to make ultra narrow mzRanges (e.g. from imported mzTab peaklist), 
+                            //a more reasonable default for a HRAM instrument (~5ppm)
+              
+                            double fiveppm = (upper*5E-6);
+                            Range<Double> tempmzRange = Range.open(lower-fiveppm,upper+fiveppm);
+                            mzRange = tempmzRange;
```

Comment:
I think it is more intuitive to use Range.closed instead of Range.open. 
The tempmzRange variable is not necessary here, you can change it to a single line:
```
mzRange = Rangeclosed(lower-fiveppm,upper+fiveppm);
```